vm/hugepage/libhugetlbfs/dependency.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+# For glibc-static dependence issue on rhel8
+# Bug 1567712 - glibc-static needs to be multilib on RHEL 8
+
+this_arch=$(uname -m)
+this_distro=$(grep DISTRO /etc/motd | cut -d '-' -f 3)
+pre_distro=${this_distro%.*}
+end_distro=${this_distro##*.}
+end_distro=${end_distro// /}
+
+if grep -q "release 8.*" /etc/redhat-release; then
+        if echo $this_distro | grep -q '.n'; then
+		choose=nightly
+        else
+		choose=rel-eng
+        fi
+        min_rel=$(grep -oE "[0-9]\.[0-9]" /etc/redhat-release | cut -d. -f2)
+
+	for ((; end_distro >= 0; end_distro-- )) do
+		build_root=http://download.eng.bos.redhat.com/$choose/rhel-8/RHEL-8/RHEL-8.${min_rel}.0-${pre_distro}.${end_distro}/compose/CRB/$this_arch/os/
+		echo "Trying option one: $build_root"
+		if curl --fail --head -s $build_root >/dev/null; then
+			success=1
+			break
+		fi
+	done
+
+	((success)) || build_root=http://download.eng.bos.redhat.com/rel-eng/rhel-8/RHEL-8/latest-RHEL-8.${min_rel}.0/compose/CRB/$this_arch/os/
+
+	echo Using repo $build_root
```

Comment:
Super nitpick: maybe use quotes in this `echo`, like in the one above? For consistency more than anything.